blueprints/app/files/.template-lintrc.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
 
 module.exports = {
-  extends: 'recommended'
+  extends: 'octane'
```

Comment:
is that available in template-lint v1 already?

Reply:
Yes